transports/websocket/src/tls.rs

```diff
@@ -35,32 +36,37 @@ impl fmt::Debug for Config {
 }
 
 /// Private key, DER-encoded ASN.1 in either PKCS#8 or PKCS#1 format.
-#[derive(Clone)]
-pub struct PrivateKey(rustls::PrivateKey);
+pub struct PrivateKey<'a>(PrivateKeyDer<'a>);
```

Comment:
can we use the owned versions of these new structs? I.e keep`pub struct PrivateKey(PrivateKeyDer<'static>);`